lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -361,14 +361,30 @@ def save_tracks(self, filename, delimiter=";", sampling_width=None):
 
     def _load_from_ui(self):
         try:
-            self.tracks = import_kymotrackgroup_from_csv(
-                self._output_filename, self._kymo, self._channel
-            )
+            self.tracks = load_tracks(self._output_filename, self._kymo, self._channel)
             self._update_tracks()
             self._set_label("status", f"Loaded {self._output_filename}")
-        except (RuntimeError, IOError) as exception:
+        except (RuntimeError, ValueError, IOError) as exception:
             self._set_label("status", str(exception))
 
+    def load_tracks(self, filename, delimiter=";"):
+        """Loads tracks from a csv file into the widget
+
+        .. note::
+
+            If you cropped the kymograph prior to tracking, you will have to make sure that you crop
+            the kymograph you supply to this function in the same way.
```

Comment:
I also mention in this docs that the restrictions are the same from `load_tracks` and recommend users to read that if they want more details.

Reply:
Added!